alcs-frontend/src/app/features/board/recon-card-detail-dialog/recon-card-detail-dialog.component.scss

```diff
@@ -0,0 +1,111 @@
+@import '../../../../styles/colors';
+
+.close {
+  color: $grey;
+  margin-top: -12px;
+  margin-right: -12px;
+  display: flex;
+  justify-content: right;
+  margin-bottom: 16px;
+}
+
+.header-row {
+  display: flex;
+  align-content: center;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 4px;
+}
+
+.application-type {
+  margin-left: 16px;
+  border-radius: 10px;
+  padding: 0 14px;
+  font-size: 11px;
+  letter-spacing: 0.01em;
+  font-weight: 700;
+}
+
+.icon {
+  margin-right: 4px;
+  height: 18px;
+  width: 18px;
+  font-size: 18px;
+}
+
+.inactive-timer {
+  opacity: 0.3;
+}
+
+.card-assignee {
+  .assignee-card-name {
+    font-size: 1.2em !important;
+    font-weight: 600 !important;
+  }
+  .assignee-card-email {
+    font-size: 0.9em !important;
+  }
+}
+
+::ng-deep .move-board-menu {
```

Comment:
Can you use :host to prevent this from becoming truly global?
:host::ng-deep

Reply:
sure, I think we have exactly the same thing on application card too since recon is mostly a copy of application. I'll check there too